lib/metalsmith.js

```diff
@@ -69,7 +69,8 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
 
   // convert *.scss files to *.css
   .use(sass({
-    includePaths: ['node_modules'] // an array of paths that sass can look in when attempt to resolve @import declarations
+    quietDeps: true,
+    loadPaths: ['node_modules', 'src/stylesheets'] // an array of paths that sass can look in when attempt to resolve @import declarations
```

Comment:
Possibly something a bit strange about putting `src/stylesheets` in loadPaths, but can't think of any practical reasons not to…

I guess the alternative would be to make all of the imports of `example-init` relative? I wonder how it ever worked with node-sass 🤔

Reply:
Woops sorry, think I pressed merge as you reviewed! Yeah I was a bit confused about how this was ever working before. I did try with relative imports but still couldn't seem to get it to work and it felt a bit... messy. 🤔 Maybe worth seeing if this causes any issues before revisiting?